apps/src/lib/node/ledger/shell/init_chain.rs

```diff
@@ -127,6 +127,7 @@ where
         if let Some(config) = genesis.ethereum_bridge_params {
             tracing::debug!("Initializing Ethereum bridge storage.");
             config.init_storage(&mut self.storage);
+            self.start_ethereum_oracle_if_necessary();
```

Comment:
this shouldn't be needed anymore right? since the shell is starting it
```suggestion
```
`start_ethereum_oracle_if_necessary()` could also check if the ethereum bridge is enabled, through `EthBridgeQueries::check_status()` (or whatever it's called, forgot the method name). this way we only start the oracle if the bridge is enabled

Reply:
Yeah, I think calling it in `Shell::new` should cover all the bases.

Reply:
It's necessary in the case where we start a node for the first time and the Ethereum bridge is configured in genesis, at `Shell::new` we won't yet have written Ethereum bridge config to storage - so we need to check in `InitChain` as well (this should only ever happen once)

Reply:
will add `EthBridgeQueries::check_status()`